src/components/search/Search.tsx

```diff
@@ -59,6 +60,10 @@ export const Search: FC = observer(() => {
         }
     })
 
+    useEffect(() => {
```

Comment:
what for is this?

Reply:
`useRef` only instantiates after the DOM ha rendered, we need an effect to set the `inputRef` within the `SearchViewStore` to the actual ref that has been created after render.

Reply:
use function callback for setting this ref: like [here](https://github.com/liquidinvestigations/hoover-ui/pull/338/files#diff-84b7f54fe75b8e3933f392e999784817aedaafec3e2d65699ea2aef60b166727R85)

Reply:
MUI TextField requires `ref` so it can't work like that, lets keep your solution